Tests/ProjectSpecTests/InvalidConfigsFormatTests.swift

```diff
@@ -0,0 +1,43 @@
+import ProjectSpec
+import Testing
+import TestSupport
+import PathKit
+
+struct invalidConfigsMappingFormatTests {
```

Comment:
```suggestion
@Suite
struct invalidConfigsMappingFormatTests {
```

Do we need it? Or do you remove it for other reason?

Reply:
even if we don't apply the `@Suite`, `swift-testing` recognizes this as test suites.

> The use of the @Suite attribute is optional; types are recognized as test suites even if they do not have the @Suite attribute applied to them.

ref: https://developer.apple.com/documentation/testing/suite(_:_:)